Rakefile

```
@@ -53,7 +53,7 @@ task extract_info_list: [:patch] do
 
     {
       gameType: gameType,
-      gameName: diceBot.id,
+      gameName: diceBot.gameName,
```

```suggestion
      gameName: diceBot.name,
```

BCDice Ver2.04.00からは `DiceBot#gameName` は非推奨で、 `DiceBot#name` が新設されました。
https://github.com/bcdice/BCDice/pull/128

先のPRで間違ってましたすみません :bow:

---

ありがとうございます 
なるほど そういうわけだったのですね 修正しました